URBNSwiftyConvenience/Classes/UIViewAnchorConvenience.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 
 /// Support safe area layout guides on supported platforms
 ///
-extension UIView {
+public extension UIView {
```

Comment:
Ugh, had to [read the docs](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/AccessControl.html#//apple_ref/doc/uid/TP40014097-CH41-ID25) to ensure this was enough:
> … you can mark an extension with an explicit access level modifier (for example, private extension) to set a new default access level for all members defined within the extension. This new default can still be overridden within the extension for individual type members.